x-pack/plugins/cases/public/components/user_actions/property_actions/alert_property_actions.test.tsx

```diff
@@ -32,93 +27,101 @@ describe.skip('AlertPropertyActions', () => {
   };
 
   beforeEach(() => {
-    appMock = createAppMockRenderer();
     jest.clearAllMocks();
+    appMock = createAppMockRenderer();
   });
 
   it('renders the correct number of actions', async () => {
-    const result = appMock.render(<AlertPropertyActions {...props} />);
+    appMock.render(<AlertPropertyActions {...props} />);
 
-    expect(result.getByTestId('property-actions-user-action')).toBeInTheDocument();
+    expect(await screen.findByTestId('property-actions-user-action')).toBeInTheDocument();
 
-    userEvent.click(result.getByTestId('property-actions-user-action-ellipses'));
+    userEvent.click(await screen.findByTestId('property-actions-user-action-ellipses'));
     await waitForEuiPopoverOpen();
 
-    expect(result.getByTestId('property-actions-user-action-group').children.length).toBe(1);
-    expect(result.queryByTestId('property-actions-user-action-minusInCircle')).toBeInTheDocument();
+    expect((await screen.findByTestId('property-actions-user-action-group')).children.length).toBe(
+      1
+    );
+
+    expect(
+      await screen.findByTestId('property-actions-user-action-minusInCircle')
+    ).toBeInTheDocument();
   });
 
   it('renders the modal info correctly for one alert', async () => {
-    const result = appMock.render(<AlertPropertyActions {...props} />);
+    appMock.render(<AlertPropertyActions {...props} />);
 
-    expect(result.getByTestId('property-actions-user-action')).toBeInTheDocument();
+    expect(await screen.findByTestId('property-actions-user-action')).toBeInTheDocument();
 
-    userEvent.click(result.getByTestId('property-actions-user-action-ellipses'));
+    userEvent.click(await screen.findByTestId('property-actions-user-action-ellipses'));
     await waitForEuiPopoverOpen();
 
-    expect(result.queryByTestId('property-actions-user-action-minusInCircle')).toBeInTheDocument();
+    expect(
```

Comment:
Maybe this is redundant.

The line below already waits for `'property-actions-user-action-minusInCircle'` so if it is not in the document it will throw there.

Reply:
Good point. I will remove them.